docs/radio.rst

```diff
@@ -0,0 +1,79 @@
+Radio
+*****
+
+.. py:module:: radio
+
+The ``radio`` module allows devices to work together via simple wireless
+networks.
+
+Functions
+=========
+
+.. py:function:: send(message)
+
+    Sends a message, expressed as a string or bytes.
+
+.. py:function:: recv_bytes()
+
+    Receive the next incoming message on the message queue. Returns ``None`` if
+    there are no pending messages. Messages are returned as bytes.
+
+.. py:function:: receive(fail=False)
+
+    Works in exactly the same way as ``recv_bytes`` but messages are expressed
+    as strings. If the optional ``fail`` argument is ``True``, a ``ValueError``
+    is raised if the incoming message cannot be converted to a string. If the
+    ``fail`` argument is ``False`` (the default) such messages and associated
+    exceptions are silently ignored.
```

Comment:
Maybe we should define/say that `receive()` is equivalent to `str(recv_bytes(), 'utf8')`?